### ISSUE TITLE
Making Moment Maps -- Missing `import` statments

### ISSUE THREAD

**Issue by HeloiseS** [Deech08/whampy#10]
## Issue

There are 2 missing imports in your tutorial [this tutorial ](https://whampy.readthedocs.io/en/latest/moment_maps.html) and then it works spot on :+1: . Just add:

```
import matplotlib.pyplot as plt
import cartopy.crs as ccrs
```

## Original Issue for history
>  I am testing [this tutorial ](https://whampy.readthedocs.io/en/latest/moment_maps.html) and received the following error.
> 
> A bit of googling seems to reveal this is a cartopy dependency - I didn't install anything beyond whampy and the tutorial doesn't call for a cartopy import, so this could be it. 
> 
> ```
> >>> mean_vel = survey.moment(order=1)
> /home/fste075/anaconda3/envs/whampy_test/lib/python3.7/site-packages/astropy/table/column.py:991: RuntimeWarning: invalid value encountered in less
>   result = getattr(super(), op)(other)
> >>> fig = plt.figure()
> >>> ax = fig.add_subplot(111, projection = ccrs.Mollweide(central_longitude=0))
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
> NameError: name 'ccrs' is not defined
>```

-----
*This issue is part of the current JOSS-review of whampy ( [#1940](https://github.com/openjournals/joss-reviews/issues/1940) )*

**Comment by Deech08** [comment]
will be fixed in next commit!

**Comment by HeloiseS** [comment]
Is it a lot of hassle to make a new pip version when you've fixed most of the initial issues we flagged?

This allows me to create a completely fresh environment in conda and pip install it anew as if I was a first time user :) I find this is the a great way to shoo the bugs away.

**Comment by HeloiseS** [comment]
Also, I should have said that the last two commands seem to have an extra indent:

```
>>> fig = plt.figure()
    >>> ax = fig.add_subplot(111, projection = ccrs.Mollweide(central_longitude=0))

    >>> fig = survey.intensity_map(ax = ax, c = mean_vel, vmin = -75, vmax = 75,
                                       norm = None, cmap = "RdBu_r", colorbar = True,
                                       cbar_kwargs={"orientation":"horizontal",
                                                            "label":"Mean Velocity (km/s)"})
```

**Comment by Deech08** [comment]
I will definitely make a new pip version once I make all these changes! I will probably do so in the next day or so and will post on these issues when it is done.

**Comment by Deech08** [comment]
Fixed and incorporated into version 0.2.0a2
